portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsFields.testcase

```diff
@@ -0,0 +1,500 @@
+@component-name = "portal-web-forms-and-data-lists"
+definition {
+
+	property custom.properties = "minifier.enabled=true";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Forms";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			Navigator.openURL();
+
+			ProductMenu.gotoPortlet(
+				category = "Content &amp; Data",
+				portlet = "Forms");
+
+			FormsAdmin.teardownForms();
+
+			Page.tearDownCP();
+		}
+	}
+
+	@description = "LPS-68182 - Verify that the Field Label is blank when adding a new Field"
+	@priority = "3"
+	test AddingANewFieldCreatesABlankLabel {
```

Comment:
@leticiachiu, can we use the words `...ADefaultLabel` instead of `...ABlankLabel`?

The use case checked by this test looks for the default label (Text) and not a blank label, as you can see on lines 67 to 69. Maybe, this is old code referencing old product behavior. :wink: 

Can you please discuss with Diogo about it?

Thanks in advance.